### PR TITLE
update api definition of /peer/heartbeat and /peer/network

### PR DESCRIPTION
**Pull request by wangforthinker**
Signed-off-by: allen.wq <allen.wq@alipay.com>

### Ⅰ. Describe what this PR did
Update the api of /peer/heartbeat and /peer/network. 

### Ⅱ. Does this pull request fix one issue?

NONE.

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
no.

### CODE REVIEW

File: apis/swagger.yml [hunk 1]
```diff
@@ -655,11 +655,23 @@ paths:
         This endpoint is mainly for reporting the heart beat to supernode.
         And supernode could know if peer is alive in strem mode.
       parameters:
-        - name: "body"
-          in: "body"
-          description: "request body which contains peer info."
-          schema:
-            $ref: "#/definitions/HeartBeatRequest"
+        - name: ip
```
Comment (allencloud) [inline apis/swagger.yml:658]: Is this a Post request, rather than GET?

File: apis/swagger.yml [hunk 2]
```diff
@@ -655,11 +655,23 @@ paths:
...
+          in: query
+          required: true
+          description: "IP of peer."
+          type: "string"
```
Comment (allencloud) [inline apis/swagger.yml:662]: format should be `ipv4`

Comment (wangforthinker) [reply apis/swagger.yml:662]: In api defintion, paramter.types should in [string number boolean integer array], which could not set ipv4.

Comment (lowzj) [reply apis/swagger.yml:662]: `format: 'ipv4'` if we only support `ipv4`

Comment (wangforthinker) [reply apis/swagger.yml:662]: done.

File: apis/swagger.yml [hunk 3]
```diff
@@ -1486,31 +1499,6 @@ definitions:
         items:
           $ref: "#/definitions/Node"
 
-  HeartBeatRequest:
```
Comment (allencloud) [inline apis/swagger.yml:1489]: If the HTTP request method could be changed into `POST`, please remain the `HeartBeatRequest ` here. That could be better.

Comment (wangforthinker) [reply apis/swagger.yml:1489]: done

**Comment by allencloud** [comment]
LGTM

**Pull request merged**